src/components/MediaLink/Video.vue

```diff
@@ -32,7 +32,7 @@
           <div class="sk-circle-wrapper" id="animation">
             <bds-loading-spinner size="small" color="main"></bds-loading-spinner>
           </div>
-          <video id="blipVideo"></video>
+          <video id="blipVideo" preload="metadata"></video> 
```

Comment:
There is an extra space at the end of this line.

Reply:
Yes, I had to create an extra commit to ensure the semantic release, because I had forgotten about it.